### ISSUE TITLE
Documentation Error on nn.ctc_loss

### ISSUE THREAD

**Issue by leonardoaraujosantos** [tensorflow/tensorflow#34586]
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:
https://www.tensorflow.org/api_docs/python/tf/nn/ctc_loss

## Description of issue (what needs changing):
The logits and logits_time_major parameters are ill defined.
### Current on docs
* logits: tensor of shape [frames, batch_size, num_labels], if logits_time_major == False, shape is [batch_size, frames, num_labels].
* logits_time_major: (optional) If True (default), logits is shaped [time, batch, logits]. If False, shape is [batch, time, logits]

### Clear description
* logits: tensor of shape [time, batch_size, num_labels], if logits_time_major == True. Shape is [batch_size, frames, num_labels] if logits_time_major == False.
* logits_time_major: (optional) If True (default), logits is shaped [time, batch_size, num_labels]. If False, shape is [batch_size, time, num_labels]

### Submit a pull request?
I'm preparing a PR

**Comment by Saduf2019** [comment]
@leonardoaraujosantos 
Is this still an issue.
